OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -23,40 +23,40 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 	public class LobbyLogic : ChromeLogic, INotificationHandler<TextNotification>
 	{
 		[TranslationReference]
-		const string Add = "add";
+		const string Add = "options-slot-admin.add-bots";
 
 		[TranslationReference]
-		const string Remove = "remove";
+		const string Remove = "options-slot-admin.remove-bots";
 
 		[TranslationReference]
-		const string ConfigureBots = "configure-bots";
+		const string ConfigureBots = "options-slot-admin.configure-bots";
 
 		[TranslationReference("count")]
-		const string NumberTeams = "n-teams";
+		const string NumberTeams = "options-slot-admin.teams-count";
 
 		[TranslationReference]
-		const string HumanVsBots = "humans-vs-bots";
+		const string HumanVsBots = "options-slot-admin.humans-vs-bots";
 
 		[TranslationReference]
-		const string FreeForAll = "free-for-all";
+		const string FreeForAll = "options-slot-admin.free-for-all";
 
 		[TranslationReference]
-		const string ConfigureTeams = "configure-teams";
+		const string ConfigureTeams = "options-slot-admin.configure-teams";
 
 		[TranslationReference]
-		const string Back = "back";
+		const string Back = "button-back";
 
 		[TranslationReference]
-		const string Team = "team";
+		const string TeamChat = "button-team-chat";
 
 		[TranslationReference]
-		const string All = "all";
+		const string GeneralChat = "button-general-chat";
```

Comment:
```suggestion
		const string AllChat = "button-all-chat";
```

Use gamer lingo?

Reply:
It's called "general chat" in the hotkeys so I went for consistency.
![image](https://user-images.githubusercontent.com/1355810/207067536-05f4e6fc-d3c9-420a-8731-db3e23ca539f.png)

Reply:
oh, perhaps we should change that

Reply:
IMO "Open General Chat" makes a lot more sense than "Open All Chat" (which could be interpreted as opening more than one chat dialog at the same time).

Reply:
I also think "general chat" is better and in case this PR is not the place to change it.